lib/auth/apiserver.go

```diff
@@ -1035,11 +1044,16 @@ func (s *APIServer) registerUsingToken(auth ClientI, w http.ResponseWriter, r *h
 	// Pass along the remote address the request came from to the registration function.
 	req.RemoteAddr = r.RemoteAddr
 
-	keys, err := auth.RegisterUsingToken(req)
+	certs, err := auth.RegisterUsingToken(req)
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
-	return keys, nil
+	return &registerUsingTokenResponseJSON{
```

Comment:
Could we capture a possible regression with tests?

Reply:
Would be tough, because the regression was only if you ran old client code against an auth server with this code, and when running tests the whole code base (client and server) is on latest. Another downside of having client and server code all together in the same codebase without clean separation.

I suppose you could write a test that implements its own client rather than using our client code, and it would have broken in this case. But I think we need to move towards having less clients, not more.

The real issue here is that lib/auth/apiserver is in need of some updates, but it's going away as we convert all of the endpoints to GRPC, so I think we should focus our efforts on that conversion rather than beefing up the testing for an API we're trying to deprecate.

Reply:
Got it, fair enough.